lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -138,6 +138,8 @@ pub struct Mouse {
     /// sampled and the most recent sample is stored here. Please note, that this sample may be
     /// from a few frames back, as it is not guaranteed when we will receive the data from GPU.
     pub pointer_target_encoded:  Uniform<Vector4<u32>>,
+    /// The display objects which believe they are currently hovered by mouse.
```

Comment:
Is there a reason that we ever believe one is hovered while if it is not hovered? If so we should mention the caveats here. Or just say that this are the currently hovered items.

Reply:
The caveat is that their information may be outdated (we read what shape is under the mouse with some offset, no greater than several frames). I'll extend the comment here.